client/src/pages/checkout/CheckoutPage.tsx

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Clean up! Perhaps a run through prettier is needed on all these files? (This one definitely needs it!)

Reply:
good idea! just ran it through all the files. thank you!